### PR TITLE
HWKAPM-861 - Added 'admin', changed current code to use it instead of 'jdoe'

### PR DESCRIPTION
**Pull request by jpkrohling**

### CODE REVIEW

**Comment by jpkrohling** [comment]
@jkandasa , would you please verify this change?

File: dist/src/main/security/application-users.properties [hunk 1]
```diff
@@ -38,6 +38,7 @@
 #
 #admin=2a0923285184943425d1f53ddd58ec7a
```
Comment (jkandasa) [inline dist/src/main/security/application-users.properties]: @jpkrohling We may remove this default, line no 39

File: dist/src/main/security/application-roles.properties [hunk 2]
```diff
@@ -38,5 +38,6 @@
 #
 #admin=PowerUser,BillingAdmin,
```
Comment (jkandasa) [inline dist/src/main/security/application-roles.properties]: @jpkrohling We may remove this default, line no 39

**Comment by jpkrohling** [comment]
@jkandasa thanks for testing! I fixed the password for the admin user. Would you please re-review? I kept the other commented out lines, as I believe they are like that in Wildfly/EAP.

**Comment by jkandasa** [comment]
@jpkrohling I build(skipped tests) this PR locally and tested with username `jdoe`/`password` and `admin`/`password`, works good. however if I enable tests(`mvn install -Pitest`), I can see failures on ` Hawkular APM::Tests::OpenTracing Agent`

`Tests run: 45, Failures: 31, Errors: 2, Skipped: 1`

**Comment by jpkrohling** [comment]
Can you confirm the SHA for the code you tested? I fixed some tests which might be the same as the ones you reported (`org.hawkular.apm.tests.agent.opentracing.server.http.JavaxServletSyncServerITest`).

**Comment by objectiser** [comment]
@jpkrohling @jkandasa Just built locally running all tests and seems to work fine now.

**Comment by jkandasa** [comment]
@jpkrohling @objectiser I followed [this articale](https://gist.github.com/piscisaureus/3342247) to pull this PR locally.

I see failures on the local build. Commit SHA: a0269201e7990f78476c6b4b918418514ef445f6

Command: `mvn install -Pitest`
[Maven console log file](https://paste.fedoraproject.org/paste/FhCNNJ-QrWRjiR1xRrZj4l5M1UNdIGYhyRLivL9gydE=/raw)

**Comment by objectiser** [comment]
retest this please

**Comment by jkandasa** [comment]
@objectiser @jpkrohling I see test failure on master branch too.
```
Failed tests: 
  ApacheHttpClientITest.testHttpClientWithResponseHandlerGET:139->testHttpClientWithResponseHandler:197
  ApacheHttpClientITest.testHttpClientWithResponseHandlerGETBadURL:144->testHttpClientWithResponseHandler:197
  ApacheHttpClientITest.testHttpClientWithResponseHandlerGETWithQS:149->testHttpClientWithResponseHandler:197
  ApacheHttpClientITest.testHttpClientWithResponseHandlerPOST:159->testHttpClientWithResponseHandler:197
  ApacheHttpClientITest.testHttpClientWithResponseHandlerPUT:154->testHttpClientWithResponseHandler:197
  ApacheHttpClientITest.testHttpClientWithoutResponseHandlerGET:56->testHttpClientWithoutResponseHandler:125
  ApacheHttpClientITest.testHttpClientWithoutResponseHandlerGETBadURL:61->testHttpClientWithoutResponseHandler:125
  ApacheHttpClientITest.testHttpClientWithoutResponseHandlerGETNoData:71->testHttpClientWithoutResponseHandler:125
  ApacheHttpClientITest.testHttpClientWithoutResponseHandlerGETWithFault:86->testHttpClientWithoutResponseHandler:125
  ApacheHttpClientITest.testHttpClientWithoutResponseHandlerGETWithQS:66->testHttpClientWithoutResponseHandler:125
  ApacheHttpClientITest.testHttpClientWithoutResponseHandlerPOST:81->testHttpClientWithoutResponseHandler:125
  ApacheHttpClientITest.testHttpClientWithoutResponseHandlerPUT:76->testHttpClientWithoutResponseHandler:125
  JBossRESTEasyClientITest.testJaxRSClientGET:52->processResponse:99
  JBossRESTEasyClientITest.testJaxRSClientGETWithFault:88->processResponse:99
  JBossRESTEasyClientITest.testJaxRSClientGETWithQS:61->processResponse:99
  JBossRESTEasyClientITest.testJaxRSClientPOST:70->processResponse:99
  JBossRESTEasyClientITest.testJaxRSClientPUT:79->processResponse:99
  JavaHttpURLConnectionClientITest.testHttpURLConnectionConnectGET:43->testHttpRequestConnect:88->sendRequest:123->verifyTrace:162
  JavaHttpURLConnectionClientITest.testHttpURLConnectionConnectGETWithFault:58->testHttpRequestConnect:88->sendRequest:123->verifyTrace:162
  JavaHttpURLConnectionClientITest.testHttpURLConnectionConnectPOST:79->testHttpRequestConnect:88->sendRequest:123->verifyTrace:162
  JavaHttpURLConnectionClientITest.testHttpURLConnectionConnectPUT:69->testHttpRequestConnect:88->sendRequest:123->verifyTrace:162
  JavaHttpURLConnectionClientITest.testHttpURLConnectionGetStatusCodeGET:48->testHttpRequestGetStatusCode:92->sendRequest:123->verifyTrace:162
  JavaHttpURLConnectionClientITest.testHttpURLConnectionInputStreamGET:53->sendRequestUsingInputStream:157->verifyTrace:162
  JavaHttpURLConnectionClientITest.testHttpURLConnectionInputStreamPOST:84->sendRequestUsingInputStream:157->verifyTrace:162
  JavaHttpURLConnectionClientITest.testHttpURLConnectionInputStreamPUT:74->sendRequestUsingInputStream:157->verifyTrace:162

Tests run: 45, Failures: 25, Errors: 0, Skipped: 1

```

**Comment by jkandasa** [comment]
@jpkrohling @objectiser Looks like I have some issues with my local envirnment setup. Jenkins says green signal. I have to check my environment setup. We can go with Jenkins result.

**Comment by objectiser** [comment]
@jkandasa Thanks for testing - it runs fine in jenkins so will merge for now and check out your issues separately.

**Comment by objectiser** [comment]
@jpkrohling Thanks!

**Pull request merged**